maxim/logger/langchain/utils.py

```
@@ -344,8 +356,12 @@ def parse_token_usage_for_result(result: LLMResult):
     total_tokens = 0
     generations = result.generations
     if generations is not None:
-        for _, generation in enumerate(generations):
+        for _, generation in enumerate(generations):            
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)